### ISSUE TITLE
Random list scroll flickering

### ISSUE THREAD

**Issue by knoopx** [airdcpp-web/airdcpp-webclient#253]
### Current UI and client versions:

AirDC++w 2.1.0 x86_64
Web UI version 2.1.0
Chrome 59.0.3071.115

### Operating system:

macOS Sierra

### Steps to reproduce the issue:

Scrolling over the User/File list causes not only the list itself to randomly flicker but also the right-side navigation bar. This does not happen on search results list.

**Comment by maksis** [comment]
Yeah, I've noticed some flickering as well... the current way of managing view data is old and somewhat ugly and it has been a while since I touched it. 

I can't see how the list views could cause the whole UI to flicker though. Does it happen with other browsers as well?
